### PR TITLE
Add banner and modal notifying users of our cookies policy

### PR DESCRIPTION
**Pull request by sraby**
This PR adds a banner that notifies user of our cookies policy and opens up a modal that explains more details about it.

NOTE: this new changes makes use of the react `localStorage` feature. In order to reset your local storage, you need to re-open localhost in an incognito window.

### CODE REVIEW

File: src/styles/_custom.scss [hunk 1]
```diff
@@ -304,3 +304,40 @@ div.is-rounded img {
...
+  }
+
+  .link {
+    display: inline;
+  }
+
+  .button {
+    width: 14px;
+    height: 14px;
+  }
```
Comment (austensen) [inline src/styles/_custom.scss:325]: I don't think this is needed? On my end all the text is squeezed 
image

File: src/styles/_custom.scss [hunk 2]
```diff
@@ -304,3 +304,40 @@ div.is-rounded img {
...
+  bottom: 0;
+  width: 100%;
+  border-top: 1px solid $justfix-black;
+
+  img {
+    filter: invert(1) contrast(0.7);
+  }
+
+  .link {
+    display: inline;
```
Comment (austensen) [inline src/styles/_custom.scss:319]: I think this also needs `cursor: pointer`

File: src/components/cookies-banner.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                  marketing purposes. This data allows us to improve our tools
+                  to give you the best experience.
+                </Trans>
+              </p>
+              <div className="buttons is-centered">
+                <button
+                  className="button is-primary"
+                  role="button"
+                  aria-label="close"
+                  onClick={() => setModalActive(false)}
```
Comment (austensen) [inline src/components/cookies-banner.tsx:89]: more of a design question - but to me it feels like clicking this "got it" button should also close the banner (`setBannerClosed(true);`)

**Review by austensen — APPROVED** [review]
This looks great! Added just a few minor comments

File: src/styles/_custom.scss (hunk 1 above)
Comment (sraby) [reply src/styles/_custom.scss:325]: oops!

File: src/styles/_custom.scss (hunk 2 above)
Comment (sraby) [reply src/styles/_custom.scss:319]: eeek! yep

**Pull request merged**